projects/tinkerbell/pbnj/Makefile

```diff
@@ -0,0 +1,53 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+GIT_TAG?=$(shell cat GIT_TAG)
+BMCLIB_GIT_COMMIT=$(shell cat BMCLIB_GIT_COMMIT)
+IPMITOOL_CLONE_URL=$(strip $(call GET_CLONE_URL,ipmitool,ipmitool))
+BMCLIB_CLONE_URL=$(strip $(call GET_CLONE_URL,bmc-toolbox,bmclib))
+IPMITOOL_GIT_TAG=$(shell cat IPMITOOL_GIT_TAG)
+GOLANG_VERSION?="1.16"
+
+REPO=pbnj
+REPO_OWNER=tinkerbell
+
+FIX_BMCLIB_TARGETS=$(REPO)/bmclib
+
+BASE_IMAGE_NAME?=eks-distro-minimal-base-glibc
+
+BINARY_TARGET_FILES=pbnj
+SOURCE_PATTERNS=.
+
+IMAGE_BUILD_ARGS=IPMITOOL_CLONE_URL IPMITOOL_GIT_TAG
+IMAGE_USERADD_USER_ID=10001
+IMAGE_USERADD_USER_NAME=pbnj
+
+include $(BASE_DIRECTORY)/Common.mk
+
+FETCH_BINARIES_TARGETS=eksa/grpc-ecosystem/grpc-health-probe
+
+$(GO_MOD_DOWNLOAD_TARGETS): $(FIX_BMCLIB_TARGETS)
+
+$(call IMAGE_TARGETS_FOR_NAME, pbnj): $(call FULL_FETCH_BINARIES_TARGETS, $(FETCH_BINARIES_TARGETS)) pbnj-useradd/images/export
+
+# PBNJ had a dependency on an older commit of bmclib module, that
+# had an LGPL dependency (gebn/bmc). This was removed in 
+# https://github.com/bmc-toolbox/bmclib/pull/257. However bmclib
+# also bumped the versions of some logging libraries like logr,
+# zap, logusr so directly using the PR's commit caused issues
+# with other go mods in PBNJ. So cloning bmclib and patching it
+# with that PR's changes
+$(FIX_BMCLIB_TARGETS): | $(GIT_PATCH_TARGETS)
```

Comment:
I dont like this, but I can let it slide.  It seems like we can potentially remove this later on anyway after pnbj releases again with the updated dep?

Reply:
Yeah I also hate having to do this but it works. I can raise an issue with PBNJ to update to the latest bmclib, pointing to this PR.

Reply:
@abhay-krishna we could bring this up in the next tinkerbell community meeting as well. If you do end up creating an issue, please share the link.